OptimizelySDK/OptlyConfig/OptimizelyConfigService.cs

```diff
@@ -31,16 +33,39 @@ public OptimizelyConfigService(ProjectConfig projectConfig)
             {
                 return;
             }
+            var events = GetEvents(projectConfig);
+            var attributes = GetAttributes(projectConfig);
+            var audiences = GetAudiences(projectConfig).Select(aud => new OptimizelyAudience(aud.Id, aud.Name, aud.Conditions)).ToArray<OptimizelyAudience>();
             var experimentMap = GetExperimentsMap(projectConfig);
             var featureMap = GetFeaturesMap(projectConfig, experimentMap);
             OptimizelyConfig = new OptimizelyConfig(projectConfig.Revision,
                 projectConfig.SDKKey,
                 projectConfig.EnvironmentKey,
+                attributes,
```

Comment:
won't this break existing constructor?

Reply:
No actually attributes sdkKey and env key are all in the new constructor.